<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EncryptedTestEditor</class>
 <widget class="QMainWindow" name="EncryptedTestEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>713</width>
    <height>431</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Encrypted Text Editor</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="1">
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/img/img/file.png</normaloff>:/img/img/file.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Encrypt</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_3">
        <item row="0" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QTextEdit" name="txtEncTextInput">
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string>Enter Yout Text Here</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="1" column="0" colspan="2">
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLineEdit" name="txtEncKey">
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string>Enter Key Value In Here</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnEncript">
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Encrypt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0" colspan="2">
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <widget class="QTextEdit" name="txtEncTextOutput">
            <property name="enabled">
             <bool>true</bool>
            </property>
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Encrypted Text</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="icon">
        <iconset resource="resource.qrc">
         <normaloff>:/img/img/file2.png</normaloff>:/img/img/file2.png</iconset>
       </attribute>
       <attribute name="title">
        <string>Decrypt</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout_4">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLineEdit" name="txtDecKey">
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>11</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string>Enter Key Value In Here</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btnDecript">
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Decrypt</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="0" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QTextEdit" name="txtDecTextInput">
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="placeholderText">
             <string>Enter Encrypted Text</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <widget class="QTextEdit" name="txtDecTextOutput">
            <property name="font">
             <font>
              <family>Montserrat</family>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="readOnly">
             <bool>true</bool>
            </property>
            <property name="placeholderText">
             <string>Decrypted Text</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>713</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew"/>
    <addaction name="actionOpenText"/>
    <addaction name="actionOpenEncryptedText"/>
    <addaction name="separator"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionEncryptText"/>
    <addaction name="actionDecryptText_2"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionFonts"/>
    <addaction name="actionFontColor"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
   </widget>
   <widget class="QMenu" name="menuAbout">
    <property name="title">
     <string>About</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuAbout"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpenText"/>
   <addaction name="actionOpenEncryptedText"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionEncryptText"/>
   <addaction name="actionDecryptText_2"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionFonts"/>
   <addaction name="actionFontColor"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionClear"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/add.png</normaloff>:/img/img/add.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/broom.png</normaloff>:/img/img/broom.png</iconset>
   </property>
   <property name="text">
    <string>Clear</string>
   </property>
   <property name="toolTip">
    <string>Clear</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/paste (2).png</normaloff>:/img/img/paste (2).png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
   <property name="toolTip">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/paste.png</normaloff>:/img/img/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
   <property name="toolTip">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/cut.png</normaloff>:/img/img/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
   <property name="toolTip">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionOpenText">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/foldertext.png</normaloff>:/img/img/foldertext.png</iconset>
   </property>
   <property name="text">
    <string>Open Text</string>
   </property>
   <property name="toolTip">
    <string>Open Text</string>
   </property>
  </action>
  <action name="actionOpenEncryptedText">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/folder.png</normaloff>:/img/img/folder.png</iconset>
   </property>
   <property name="text">
    <string>Open Encrypted Text</string>
   </property>
   <property name="toolTip">
    <string>Open Encrypted Text</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/save.png</normaloff>:/img/img/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/save2.png</normaloff>:/img/img/save2.png</iconset>
   </property>
   <property name="text">
    <string>SaveAs</string>
   </property>
   <property name="toolTip">
    <string>SaveAs</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/cancel.png</normaloff>:/img/img/cancel.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
   <property name="toolTip">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionEncryptText">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/file.png</normaloff>:/img/img/file.png</iconset>
   </property>
   <property name="text">
    <string>Encrypt Text</string>
   </property>
   <property name="toolTip">
    <string>Encrypt Text</string>
   </property>
  </action>
  <action name="actionDecryptText_2">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/file2.png</normaloff>:/img/img/file2.png</iconset>
   </property>
   <property name="text">
    <string>Decrypt Text</string>
   </property>
   <property name="toolTip">
    <string>Decrypt Text</string>
   </property>
  </action>
  <action name="actionFonts">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/font.png</normaloff>:/img/img/font.png</iconset>
   </property>
   <property name="text">
    <string>Fonts</string>
   </property>
   <property name="toolTip">
    <string>Fonts</string>
   </property>
  </action>
  <action name="actionFontColor">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/color-picker.png</normaloff>:/img/img/color-picker.png</iconset>
   </property>
   <property name="text">
    <string>Font Color</string>
   </property>
   <property name="toolTip">
    <string>Font Color</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/undo.png</normaloff>:/img/img/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
   <property name="toolTip">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/redo.png</normaloff>:/img/img/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
   <property name="toolTip">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/img/img/broom.png</normaloff>:/img/img/broom.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
   <property name="toolTip">
    <string>About</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
